submissions/ORybka/a-tiny-js-world/index.js

```diff
@@ -62,6 +62,6 @@ const objects = [man, woman, dog, cat, catWoman];
 
 objects.forEach(el => {
    const objFriends = el.friends.join(', ')
-   const objKeys = [`${el.species}`, `<strong>${el.name}</strong>`, `${el.gender}`, `${el.legs}`, `${el.hands}`, `<em>${el.saying}</em>`, objFriends];
```

Comment:
I wonder where this approach comes from? I mean `` `${el.species}` `` instead of plain `el.species`.
Can you please tell me that story?

Reply:
But I fixed it 😅 
In this case, I suppose, I concentrated my attention to HTML tags (e.g. `<strong>${el.name}</strong>`) and haven't thought about other items. My mistake 😞  I'm afraid, I can't reveal this secret.